server/models/Conversation.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+const { ObjectId } = Schema;
+
+const ConversationSchema = new Schema({
```

Comment:
Date fields for both of these models (like Kevin used in Project) would be helpful.